templates/assets/css/vlabs-style.css

```diff
@@ -1,180 +1,297 @@
-    text-align: justify;
+  font-family: "Raleway", sans-serif;
+  color: #000000;
+  text-align: justify;
 }
 
 /* to fix the  extended lines*/
 .markdown-body table tr {
-    border-top: 0;
+  border-top: 0;
 }
 
-h1, h2, h3 {
-    color: #2C99CE;
+h1,
+h2,
+h3 {
+  color: #2c99ce;
 }
 
 h2 {
-    padding-top: 2rem;
-    padding-bottom: 1rem;
+  padding-top: 2rem;
+  padding-bottom: 1rem;
 }
 
 h3 {
-    font-size: 1.1rem;
-    color: #333333;
-    padding-top: 1rem;
-    text-decoration: underline;
+  font-size: 1.1rem;
+  color: #333333;
+  padding-top: 1rem;
+  text-decoration: underline;
 }
 
-
 /* for ds experiments - video iframes */
 iframe {
-    width: 100%;
-    height: calc(100vw/3);
+  width: 100%;
+  height: calc(100vw / 3);
 }
 
-
 .simulation-container {
-    position: relative;
-    overflow: hidden;
-    width: 100%;
-    padding-top: 56.25%;
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  padding-top: 56.25%;
 }
 
 .responsive-iframe {
-    position: absolute;
-    top: 0;
-    left: 0;
-    bottom: 0;
-    right: 0;
-    width: 100%;
-    height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
 }
 
 .visible {
-    width: max-content;
+  width: max-content;
 }
 
 .hidden {
-    width: 0px;
-    border-width: 0px;
-    padding: 0px;
+  width: 0px;
+  border-width: 0px;
+  padding: 0px;
+}
+
+/* Style the buttons that are used to open and close the accordion panel */
+.accordion {
+  display: none;
+  margin-left: 20px;
+  color: #337ab7;
+  text-decoration: underline;
+  text-align: right;
+}
+
+/* Add a background color to the button if it is clicked on (add the .active class with JS), and when you move the mouse over it (hover) */
+/* .active,
+.accordion:hover {
+} */
+
+/* Style the accordion panel. Note: hidden by default */
+.panel {
+  display: none;
+  background-color: #eee;
+  color: #444;
+  padding: 18px;
+  width: 100%;
+  text-align: left;
+  border: none;
+  outline: none;
+  transition: 0.4s;
+}
+
+/* Customize the label (the container) */
+.container {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 22px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+/* Hide the browser's default checkbox */
+.container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+/* Create a custom checkbox */
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 25px;
+  width: 25px;
+  background-color: #eee;
+}
+
+/* On mouse-over, add a grey background color */
+.container:hover input ~ .checkmark {
+  background-color: #ccc;
+}
+
+/* When the checkbox is checked, add a blue background */
+.container input:checked ~ .checkmark {
+  background-color: #2196f3;
+}
+
+/* Create the checkmark/indicator (hidden when not checked) */
+.checkmark:after {
```

Comment:
Why do we need this?

Reply:
I have used checkmarks with custom css sir, the css here accounts for the working of checkmarks, I can maybe refactor this once, our css cdn is up. we can have our own checkmark component there.